PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/java/com/microsoft/azure/toolkit/intellij/azure/sdk/buildtool/ServiceBusReceiverAsyncClientCheck.java

```diff
@@ -0,0 +1,34 @@
+package com.microsoft.azure.toolkit.intellij.azure.sdk.buildtool;
+
+import com.intellij.codeInspection.*;
+import com.intellij.psi.*;
+import org.jetbrains.annotations.NotNull;
+
+
+/**
+ * This class extends the LocalInspectionTool to check for the use of ServiceBusReceiverAsyncClient
+ * in the code and suggests using ServiceBusProcessorClient instead.
+ */
+public class ServiceBusReceiverAsyncClientCheck extends LocalInspectionTool {
```

Comment:
Need to add this rule to the readme too?

Reply:
done